docs/source/tutorial/tutorial_2.mdx

```diff
@@ -588,17 +594,15 @@ You'll see a warning in GraphiQL:
 
 ![variable dollar sign id of type ID was used in position expecting type ID exclamation point](images/graphiql_nullable_conflict.png)
 
-This bit can be VERY confusing for Swift developers in particular, because GraphQL's assumptions about nullability are essentially the opposite of Swift's. 
+This bit can be confusing, because GraphQL's assumptions about nullability are different from Swift's: 
 
-In Swift, if you don't annotate a property's type with either a question mark or an exclamation point, that property is non-nullable. 
+* In Swift, if you annotate a property's type with either a question mark or an exclamation point, that property is nullable. 
```

Comment:
This is not correct. 

It's nullable if there's a question mark. It's implicitly unwrapped if it's got an exclamation point. In theory you can set either to null but calling it "nullable" will make Swift devs go wtf.

I'd like to go back to the original explanation on these two because I think for Swift developers this framing will make a lot more sense.

Maybe "That property can be set to null".